tests/unittests.py

```diff
@@ -0,0 +1,310 @@
+import pytest
+import sourmash
+import pickle
+import numpy as np
+import pandas as pd
+import csv
+import os
+import sys
+from scipy.sparse import csc_matrix
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from  srcs.utils import *
```

Comment:
IIRC, PEP8 says that `import *` isn't recommended as it can make it harder to track down names. Eg. which functions are in `src.utils` and which are in `src.hypothesis_recovery_src`. Perhaps make it explicit via eg. `srcs.utils.<func>` instead of in a doc string/commend (line 14)